admin-dev/themes/new-theme/scss/components/_updatable_input.scss

```diff
@@ -0,0 +1,36 @@
+.ps-updatable-input-wrapper {
+  position: relative;
+  .ps-number-spinner {
+    right: 34px;
+  }
+  .check-button {
```

Comment:
It's strange to have no new lines between these scopes, I thought stylelint had a rule for this, is it a really old branch?

Reply:
nope, its new